nesta/packages/arxiv/tests/test_collect_arxiv.py

```diff
@@ -513,4 +514,12 @@ def test_multiple_results_are_returned_for_multinationals(self):
         article = Article(id=3)
 
         links = create_article_institute_links(article, ['a', 'b', 'c'], score)
-        assert len(links) == 3
\ No newline at end of file
+        assert len(links) == 3
+
+
+@mock.patch('nesta.packages.arxiv.collect_arxiv.db_session')
+def test_all_article_ids(mocked_session):
+    Tmp = namedtuple('Tmp',['id'])
+    mocked_session.return_value.__enter__.return_value.query.return_value.limit.return_value = [Tmp(i) for i in range(0, 131)]*83
```

Comment:
maybe consider:
`mocked_session().__enter__().query().limit.return_value`

Reply:
didn't realise that the nice syntax existed!